### PR TITLE
Edit: Fix rename for playlists

### PR DESCRIPTION
**Pull request by Mikanbu**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

Closes [#603](https://code.videolan.org/videolan/vlc-ios/issues/603).

### CODE REVIEW

File: Sources/EditController.swift [hunk 1]
```diff
@@ -224,36 +224,52 @@ extension EditController: EditToolbarDelegate {
...
+                return
+            }
+
+            let mlObject = self?.model.anyfiles[indexPath.row]
+
+            if let media = mlObject as? VLCMLMedia {
+                media.updateTitle(text)
+            } else if let playlist = mlObject as? VLCMLPlaylist {
+                playlist.updateName(text)
+            }
```
Comment (Mikanbu) [inline Sources/EditController.swift]: Not a big fan of the duplication here and above for the name/title.

Suggestions welcome! 🎉

Comment (carolanitz) [reply Sources/EditController.swift]: you could have a wrapper for playlist that is called updateTitle and calls updateName 🤷‍♀️

Comment (Mikanbu) [reply Sources/EditController.swift]: I agree that this isn't great, for now I will add the assertion as you suggested!

**Comment by Mikanbu** [comment]
Rebased to master.

**Pull request merged**